spec/core_functions/selector/extend/simple/pseudo/selector/match.hrx

```diff
@@ -6,40 +6,79 @@ they're structurally equal.
 
 <===>
 ================================================================================
-<===> unprefixed/class/equal/input.scss
+<===> unprefixed/is/class/equal/input.scss
+a {b: selector-extend(":is(c d.e, f g)", ":is(c d.e, f g)", "h")}
+
+<===> unprefixed/is/class/equal/output.css
+a {
+  b: :is(c d.e, f g), h;
+}
+
+<===>
+================================================================================
+<===> unprefixed/is/class/unequal/name/input.scss
+a {b: selector-extend(":is(c d.e, f g)", ":-pfx-is(c d.e, f g)", "h")}
+
+<===> unprefixed/is/class/unequal/name/output.css
+a {
+  b: :is(c d.e, f g);
+}
+
+<===>
+================================================================================
+<===> unprefixed/is/class/unequal/argument/input.scss
+a {b: selector-extend(":is(c d.e, f g)", ":is(d, g)", "h")}
+
+<===> unprefixed/is/class/unequal/argument/output.css
+a {
+  b: :is(c d.e, f g);
+}
+
+<===>
+================================================================================
+<===> unprefixed/is/class/unequal/has_argument/input.scss
+a {b: selector-extend(":is(c d.e, f g)", ":is", "h")}
+
+<===> unprefixed/is/class/unequal/has_argument/output.css
+a {
+  b: :is(c d.e, f g);
+}
```

Comment:
Missing newline here seems to break ruby spec runner again:
https://github.com/sass/libsass/pull/3155/checks?check_run_id=2634215399

```diff
  1) Failure:
SassSpec::Test#test__sass-spec/spec/core_functions/selector/extend/simple/pseudo/selector/match/unprefixed/is/class/unequal/has_argument [/home/runner/work/libsass/libsass/sass-spec/lib/sass_spec/test.rb:267]:
expected did not match output.
--- expected
+++ actual
@@ -1,3 +1,4 @@
 "a {
   b: :is(c d.e, f g);
-}"
+}
+"
```